decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative('nameable')
+
```

Comment:
- [ OPIONAL ] I think here it would be better if you called the namable here as  attr_accessor like this: 

>  attr_accessor :nameable